### PR TITLE
Extend NASA GrIS regions into ocean

### PR DESCRIPTION
**Pull request by alexolinhager**
This PR introduces manually modified versions of the NASA GrIS in `geometric_data/landice/region` (`eastCentralGreenland1`, etc.), so that each region now extends into the ocean to the continental shelf. The boundaries of some regions were also modified so one region flows into each fjord. For now, the PR adds the newly modified regions without replacing the originals in case existing analyses are dependent on these regions.

### CODE REVIEW

**Comment by alexolinhager** [comment]
Thanks @cbegeman, that's a good suggestion. I just pushed a commit with a new naming convention

**Comment by xylar** [comment]
@alexolinhager, I see that your features aren't using the geojson formatting that is standard here, you're using a more compact format that isn't as easy to read.  Could you follow the second procedure here to merge them all into a single feature collection and then split them back out into individual features to clean things up?
https://mpas-dev.github.io/geometric_features/main/adding_features.html

**Comment by xylar** [comment]
The important bit is:
``` python
# split the feature collection back into individual features within
# ./geometric_data to clean things up
gf.split(fc)
```
It looks like you have the tags in the database and all that.

File: docs/aggregation.rst [hunk 1]
```diff
@@ -88,16 +90,26 @@ Amundsen Sea Deep, Eastern Ross Sea Shelf, Eastern Ross Sea Deep,
...
 ISMIP6 Greenland North Shelf, ISMIP6 Greenland North West Shelf,
 ISMIP6 Greenland South East Shelf, ISMIP6 Greenland South West Shelf.
 
+NASA Greenland Regions
+~~~~~~~~~~~~~~~~~
+
+The function :py:func:`geometric_features.aggregation.landice.nasa_greenland`
+aggregates regions of Greenland drainage basins taken from NASA: 
+(https://earth.gsfc.nasa.gov/cryo/data/polar-altimetry/antarctic-and-greenland-drainage-systems)
+The drainage basins have been altered to extend to the continental shelf. Boundaries between basins
```
Comment (matthewhoffman) [inline docs/aggregation.rst:109]: Is this suggestion correct?
```suggestion
The drainage basins have been altered to extend to the continental shelf break. Boundaries between basins
```

File: docs/aggregation.rst [hunk 2]
```diff
@@ -88,16 +90,26 @@ Amundsen Sea Deep, Eastern Ross Sea Shelf, Eastern Ross Sea Deep,
...
 
-The function :py:func:`geometric_features.aggregation.ocean.greenland`
+The function :py:func:`geometric_features.aggregation.ocean.ismip6_greenland`
 aggregates regions of the continental shelves around Greenland based on the
 ISMIP6 regions: ISMIP6 Greenland Central East Shelf,
 ISMIP6 Greenland Central West Shelf, ISMIP6 Greenland North East Shelf,
 ISMIP6 Greenland North Shelf, ISMIP6 Greenland North West Shelf,
 ISMIP6 Greenland South East Shelf, ISMIP6 Greenland South West Shelf.
 
+NASA Greenland Regions
```
Comment (matthewhoffman) [inline docs/aggregation.rst:103]: Can you update the descriptions for "NASA Greenland Regions" and "ISMIP6 Greenland Regions" to indicate if each is intended for use by ocean or ice sheet applications or both?

File: geometric_features/aggregation/__init__.py [hunk 3]
```diff
@@ -51,9 +51,12 @@ def get_aggregator_by_name(region_group):
                'Ocean Subbasins': {'prefix': 'oceanSubbasins',
                                    'date': '20201123',
                                    'function': subbasins},
-               'Greenland Regions': {'prefix': 'greenlandRegions',
+               'ISMIP6 Greenland Regions': {'prefix': 'ISMIP6GreenlandRegions',
```
Comment (matthewhoffman) [inline geometric_features/aggregation/__init__.py:54]: @xylar , do you prefer that the `prefix` for this and the NASA Greenland Regions to use lower/camel case for the acronyms like is done for the "ismip6Regions" prefix?

File: geometric_features/aggregation/landice/nasa_greenland_extended.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
+def nasa_greenland(gf):
+    """
+    Aggregate NASA Greenland ice basin region extended to continental shelf.
```
Comment (matthewhoffman) [inline geometric_features/aggregation/landice/nasa_greenland_extended.py:3]:
```suggestion
    Aggregate NASA Greenland ice sheet basin regions extended to continental shelf.
```

**Review by matthewhoffman — CHANGES_REQUESTED** [review]
@alexolinhager , this looks good.  I just have a few minor suggestions.

I loaded all the new regions (green) into QGIS and overlaid the GIS outline (pink) and one of the ISMIP6 ocean regions (red) for comparison.  The background is BedMachine bed topography colored between -2000 and 0 m.  Everything looks as expected!  This will be nice to have available.


image

File: docs/aggregation.rst (hunk 1 above)
Comment (alexolinhager) [reply docs/aggregation.rst:109]: Yes that's more descriptive

File: geometric_features/aggregation/__init__.py (hunk 3 above)
Comment (alexolinhager) [reply geometric_features/aggregation/__init__.py:54]: It makes sense to me just to change the caps to make this consistent

**Review by matthewhoffman — APPROVED** [review]

**Pull request merged**